frontend/src/components/ui-kit/components/DiscountPrice/DiscountPrice.jsx

```diff
@@ -13,9 +13,23 @@ const DiscountPrice = ({ oldPrice, actualPrice }) => {
       >
         {oldPrice} грн
       </p>
-      <p className={styles['actual-price']}>
+      <p
+        className={
+          styles['actual-price'] +
+          ' ' +
+          `${isActive === 'NO_ACTIVE' ? styles.inactive : ''}`
+        }
+      >
         {actualPrice}
-        <span className={styles.currency}>грн</span>
+        <span
+          className={
+            styles.currency +
+            ' ' +
+            `${isActive === 'NO_ACTIVE' ? styles.inactive : ''}`
```

Comment:
You always have the same color here as in the p tag, so why do you need these conditions?

Reply:
p tag have two colors - one for active card and another, if card is inactive

Reply:
I'm talking about the span, it has the same colors as the p, it can all be done through css

Reply:
Done